x/ethbridge/types/claim.go

```diff
@@ -11,6 +11,8 @@ import (
 )
 
 type EthBridgeClaim struct {
+	Chain            string          `json:"chain"`
+	Contract         EthereumAddress `json:"contract_address"`

```

Comment:
is this BridgeContractAddress or TokenContractAddress?

Reply:
https://github.com/swishlabsco/peggy/pull/4/commits/be6ce9d49dac7195bbcf2bf3b5260ebd2afc5715